src/frontends/pytorch/src/op_table.cpp

```diff
@@ -862,6 +862,7 @@ const std::unordered_map<std::string, CreatorFunction> get_supported_ops_fx() {
         {"aten.hardtanh.default", op::translate_hardtanh},
         {"aten.hardtanh_.default", op::inplace_op<op::translate_hardtanh>},
         {"aten.index.Tensor", op::translate_index_fx},
+        {"aten._unsafe_index.Tensor", op::translate_index_fx},
```

Comment:
please add layer tests for this similar to `aten.index.Tensor`: https://github.com/openvinotoolkit/openvino/blob/master/tests/layer_tests/pytorch_tests/test_index_tensor.py

Reply:
Done, please check